app/controllers/admin/articles_controller.rb

```diff
@@ -45,10 +45,13 @@ def update
       article.boosted_additional_articles = article_params[:boosted_additional_articles].to_s == "true"
       article.boosted_dev_digest_email = article_params[:boosted_dev_digest_email].to_s == "true"
       article.user_id = article_params[:user_id].to_i
-      article.second_user_id = article_params[:second_user_id].to_i
-      article.third_user_id = article_params[:third_user_id].to_i
-      article.update!(article_params)
-      render body: nil
+      article.co_author_ids = article_params[:co_author_ids].split(",").map(&:strip)
+      if article.save
+        flash[:success] = "Article saved!"
+      else
+        flash[:danger] = article.errors_as_sentence
+      end
+      redirect_to admin_article_path(article.id)
```

Comment:
These changes were necessary for a few reasons:
- if `article.update!(article_params)` is used here, the article gets overwritten during `#update`, while using `article.save` updates the article in as expected.
- in order to get the flash messages and redirect to work in the `#update` action, I had to remove the green highlighting that occurred on submit in the form in `_individual_article.html.erb` and add `local: true`. Prior to these changes, it did not appear that errors were rendering for Articles at all and no redirects took place. 😱